tests/test_pandas.py

```diff
@@ -464,14 +465,15 @@ def test_crosstab() -> None:
 def test_arrow_dtype() -> None:
     pytest.importorskip("pyarrow")
 
-    import pyarrow as pa
+    if sys.version_info < (3, 11):
+        import pyarrow as pa
```

Comment:
Is this because pa is available but didn't work?

Reply:
No, it's because we don't want the type checker to do the type checking here with python 3.11, since pyarrow is not available yet.

Reply:
I see a merged PR in the pyarrow repo that suggests 3.11 is supported? 

https://github.com/apache/arrow/pull/14499